include/swift/AST/Import.h

```diff
@@ -450,6 +451,8 @@ struct ImportedModuleDesc {
   }
 };
 
+using ImportedModuleDesc = AttributedImport<ImportedModule>;
```

Comment:
Is this vestigial (possibly for ease of making changes) or intentional?

Reply:
That was temporary; it was removed in a later commit.